etna/records/tests/test_iaid.py

```diff
@@ -0,0 +1,16 @@
+from django.test import TestCase
+from django.urls import resolve
+
+
+class TestiaidFromats(TestCase):
```

Comment:
```suggestion
class TestIaidFormats(TestCase):
```

Reply:
Could we have Test case for iaid=A13530124, C2341693, D431198, F257629 as per the ticket.
Also add N14562581

Reply:
Yes , I also wanted to do that, but I found it hard to give a good name to these tests, and I don't want to put them all in a for loop also, can you please suggest the names for those tests?

Reply:
As discussed, lets go with your suggestion.